test/kpod_images.bats

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bats
+
+load helpers
+
+IMAGE="alpine:latest"
```

Comment:
This value isn't used in this file.  Maybe change it to `debian:6.0.10` instead, and use it?